eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/RecordId.scala

```diff
@@ -0,0 +1,24 @@
+package com.evolutiongaming.kafka.journal.eventual.cassandra
+
+import cats.Monad
+import cats.effect.std.UUIDGen
+import cats.syntax.all.*
+import com.evolutiongaming.scassandra.*
+
+final case class RecordId(value: String) extends AnyVal {
+  override def toString: String = value
+}
+
+object RecordId {
+  def unsafe(value: String): RecordId = new RecordId(value)
```

Comment:
I don't think there is anything unsafe about this method - it appears to be redundant if you already have `apply`. I'd remove it.

Reply:
I'd keep it with its name. `unsafe` means that one can put arbitrary value into `RecordId` ignoring its rules

Reply:
and one can use `RecordId#apply` from non-private case class constructor directly ;)